### DISCUSSION TITLE
寫程式很無聊嗎？vscode寵物陪你！

### DISCUSSION THREAD

**Discussion by williamyen0623** [1101-GS3073-C/Python-Community-GS3073-C#502] [General]
常常寫程式的時候感到枯燥乏味嗎？
沒關係 我們又要來讚嘆vscode的強大了
有了vscode-pet extension 寫code不再無聊，會有可愛的小動物陪著你，畫面相當療癒～

以下是vscode-pet的documention，安裝方式跟其他extension一樣，很簡單～
https://tonybaloney.github.io/vscode-pets/

<img width="857" alt="pet-in-default-explorer" src="https://user-images.githubusercontent.com/95861867/208291112-b65e1476-e03c-401d-b367-f2d5581680cd.png">

**Comment by NcuJerryKao** [comment]
哈哈哈哈 vscode 真的很常用 要是有隻寵物多好 謝啦
馬上就來安裝 這樣打code到三更半夜時 至少不會太孤單

**Comment by Ben6uo** [comment]
不會打code是唯一大問題阿 如果寵物能順便教就好了

**Comment by 32844583** [comment]
上網查了一下還可以透過 Python 自己寫一隻桌面寵物喔 !
https://medium.com/analytics-vidhya/create-your-own-desktop-pet-with-python-5b369be18868

**Comment by manipulateed** [comment]
能發現這個也太療癒了吧～會不會之後開始還能養寵物🤔

**Comment by gnsJhenJie** [comment]
跟Google Colab的貓咪和狗狗有異曲同工之妙呢！

**Comment by CLIsVeryOK** [comment]
您的来信已收到，谢谢！陈雷同济大学测绘与地理信息学院Thanks for your attention.Chen Lei
College of survey and geo-information of Tongji university

**Comment by xCk27x** [comment]
VS code extension那邊載好後
按「ctrl+shift+P」打開命令視窗
再輸入"Start pet coding session"指令
寵物就會在左下角出現了

**Comment by skyling0299** [comment]
喔喔喔喔喔這個我有用! 超級可愛~
這邊推薦大家fox這個種類，還有龍貓

**Comment by HsuChen88** [comment]
雖然我不養寵物，但在寫code時的確很需要一隻小動物陪伴，謝謝分享有趣的套件。

**Comment by RogelioKG** [comment]
好可愛😌
不知道什麼時候會有 VScode 寵物對戰的 extension
在咖啡廳 coding 的工程師狹路相逢，來一場寶可夢對決 (欸不是)

**Comment by awkward-willy** [comment]
前陣子有裝這個！也有推薦身旁的朋友安裝，但我自己太容易分心的，看他們在旁邊晃啊晃啊，就忍不住盯著看，所以後來還是決定先拿掉了，如果未來能讓他們到處跑，或定時餵食之類的，甚至因打的 code 不同而成長感覺會更有趣！

**Comment by Raywei0214** [comment]
希望可以用自己畫得像素圖之類的，像我們這種理工宅男不放一些香香油油的美少女角色，怎麼能說的過去呢。

**Comment by larry40305** [comment]
當code寫不出來的時候，看著寵物可以減少心態炸裂的機率
